# HG changeset patch
# User Yury Ivanovich <Yury.Ivanovich@linuxmail.org>
# Date 1713989256 -7200
# Parent  6a3511873cf41553089e3423cb4b28adef706307
Branding 2: Fix inclusion of pref/thunderbird-branding.js.

diff --git a/mail/branding/branding-common.mozbuild b/mail/branding/branding-common.mozbuild
--- a/mail/branding/branding-common.mozbuild
+++ b/mail/branding/branding-common.mozbuild
@@ -6,17 +6,11 @@
 
 
 @template
 def ThunderbirdBranding():
-    release_branding_dirs = ("comm/mail/branding/tb_beta", "comm/mail/branding/thunderbird")
-    if CONFIG["MOZ_BRANDING_DIRECTORY"] in release_branding_dirs:
-        JS_PREFERENCE_PP_FILES += [
-            "pref/thunderbird-branding.js",
-        ]
-    else:
-        JS_PREFERENCE_FILES += [
-            "pref/thunderbird-branding.js",
-        ]
+    JS_PREFERENCE_PP_FILES += [
+        "pref/thunderbird-branding.js",
+    ]
 
     if CONFIG["MOZ_WIDGET_TOOLKIT"] == "windows":
         FINAL_TARGET_FILES += [
             "thunderbird.VisualElementsManifest.xml",
